src/components/DetailsMetrics/DetailsMetrics.js

```diff
@@ -17,14 +17,61 @@ illegal under applicable law, and the grant of the foregoing license
 under the Apache 2.0 license is conditioned upon your compliance with
 such restriction.
 */
-import React from 'react'
+import React, { useState } from 'react'
+import { Form } from 'react-final-form'
+import { createForm } from 'final-form'
+import { cloneDeep } from 'lodash'
+
+import MetricsSelector from '../../elements/MetricsSelector/MetricsSelector'
+
+import './detailsMetrics.scss'
+
+const metricsMock = [
+  { name: 'One', checked: true, colorIndicator: '#eb4034' },
+  { name: 'Two', checked: false, colorIndicator: '#eba434' },
+  { name: 'Three long long long long long', checked: true, colorIndicator: '#c9eb34' },
+  { name: 'Four', checked: false, colorIndicator: '#46eb34' },
+  { name: 'Five', checked: true, colorIndicator: '#34ebb1' },
+  { name: 'Six', checked: false, colorIndicator: '#349beb' },
+  { name: 'Seven', checked: false, colorIndicator: '#5b34eb' },
+  { name: 'Eight', checked: false, colorIndicator: '#e534eb' }
+]
 // import { useSelector } from 'react-redux'
 
 const DetailsMetrics = () => {
   // const detailsStore = useSelector(store => store.detailsStore)
+  const [metrics, setMetrics] = useState(metricsMock)
+  const formRef = React.useRef(
+    createForm({
+      initialValues: {
+        metrics: metrics.filter(metricItem => metricItem.checked).map(metricItem => metricItem.name)
```

Comment:
display value shouldn't be used for any statements. Use `id` and `label`

Reply:
+